ansible/roles/test/tasks/ptf_runner.yml

```diff
@@ -51,5 +51,36 @@
 
 - debug: var=out.stdout_lines
 
+- name: Set default PTF log filename
+  set_fact:
+    ptf_log_file: "/root/ptf.log"
```

Comment:
Here default PTF log file is set to ```/root/ptf.log```, but all tests use ```/tmp/```.
I believe it would be better to change this default to ```/tmp/``` as well, so we have all logs in one place.

Reply:
This code is to specify where to get the PTF log and pcap files from the PTF container.

When PTF log file is not explicitly specified while calling ptf_runner.yml, PTF saves default log and pcap files to /root/ptf.log and /root/ptf.pcap on PTF container. 

If explicit log file parameter is parsed from the PTF command line, this default value will be overwritten with custom log file location.